File: results/WizardCoder/python_streamlit/counter.py
import streamlit as st

# Set initial value of counter
counter = 1

# Display counter
st.write("Counter:", counter)

# Define functions to handle button clicks
def increase():
    global counter
    counter += 1
    st.write("Counter:", counter)

def decrease():
    global counter
    if counter > 1:
        counter -= 1
        st.write("Counter:", counter)

def reset():
    global counter
    counter = 1
    st.write("Counter:", counter)

# Display buttons
st.button("Increase", on_click=increase)
st.button("Decrease", on_click=decrease)
st.button("Reset", on_click=reset)

File: results/WizardCoder/javascript_React/counter.js
import React, { useState } from'react';

function Counter() {
  const [count, setCount] = useState(1);

  const handleIncrease = () => {
    setCount(count + 1);
  };

  const handleDecrease = () => {
    if (count > 1) {
      setCount(count - 1);
    }
  };

  const handleReset = () => {
    setCount(1);
  };

  return (
    <div>
      <h1>{count}</h1>
      <div>
        <button onClick={handleIncrease}>Increase</button>
        <button onClick={handleDecrease}>Decrease</button>
        <button onClick={handleReset}>Reset</button>
      </div>
    </div>
  );
}

export default Counter;